internal/sourcer/registry_sourcer.go

```diff
@@ -54,7 +54,7 @@ func (cs catalogSource) Source(ctx context.Context, po *platformv1alpha1.Platfor
 }
 
 func (s sources) GetCandidates(ctx context.Context, po *platformv1alpha1.PlatformOperator) (bundles, error) {
-	if len(s) > 1 {
+	if len(s) != 1 {
```

Comment:
I saw a panic when there were no available catalog sources in the cluster. This is a follow-up to the changes made in #26.